daml-script/test/daml/upgrades/UpgradesTest.daml

```diff
@@ -469,6 +503,29 @@ canRemoveObserversThatAreSignatories =
     (\(alice, bob, charlie) -> ([alice, bob, charlie], [alice, bob, charlie])) -- signatories
     (\(alice, bob, charlie) -> ([alice, bob, charlie], [alice, bob])) -- observers
 
+-- TODO: This test should fail, as the ensure becomes negative on the V2. With current implementation, a V2 choice can be called on an
```

Comment:
could you please create tickets to track this and the TODO on `ensureClauseDowngradeToNoLongerValid`? or if they already exist could you please add them to the comments?

Reply:
See comment above.